Integrations/integration-ArcherRSA.yml

```diff
@@ -0,0 +1,626 @@
+        return result.Body;
+
+    }
+
+    function sendRestRequest(cmd, bodyContent, token, httpTunneling) {
+        var headers, body;
+        headers = {'Content-Type': ['application/json'], 'Accept': ['application/json']};
+        if(token) {
+            headers['Authorization'] = ['Archer session-id=' + token];
+        }
+        if(httpTunneling) {
+            headers['X-Http-Method-Override'] = ['GET'];
+        }
+        var url =  server + replaceInTemplatesAndRemove(urlDictionary[cmd], args)
+        body = JSON.stringify(bodyContent);
+        var result = http(
+            url,
+            {
+                Method: methodDictionary[cmd],
+                Headers: headers,
+                Body: body ? body : ''
+            },
+            params.insecure,
+            params.proxy
+        );
+
+        if (result.StatusCode < 200 || result.StatusCode >= 300) {
+            throw 'Request to RSA Archer ' + url + ' failed, request status code: ' + result.StatusCode + ' and Body: ' + result.Body + '.';
+        }
+        return JSON.parse(result.Body);
+    }
+
+
+    var fetchIncidentFlow = function () {
+        var cmd = 'archer-fetch-incidents';
+        var incidentRes = [];
+        var now = getLastRun();
+        console.log("NOW = " + JSON.stringify(now))
+        if(now.currentId === undefined) {
+            now = {currentId: 0}
+        }
+        console.log('command = ' + command + ', commandArgs = ' + JSON.stringify(commandArgs));
+        result = sendRequest(methodDictionary[cmd], replaceInTemplatesAndRemove(urlDictionary[cmd], args), token, commandArgs);
+        var output = parseRecordsXml(result);
+        var maxId = now.currentId;
+        console.log("Now.currentId = " + JSON.stringify(now));
+        output.forEach(function (el) {
+            var parsedId = parseInt(el.ID);
+            console.log("element: " + JSON.stringify(el) + " and parsedId = " + parsedId);
+            if(parsedId >  now.currentId) {
+                console.log("maxId = " + maxId);
+                incidentRes.push(el);
+                if(maxId < parsedId){
+                    maxId = parsedId;
+                }
+            }
+        });
+        setLastRun({currentId: maxId});
+        return (JSON.stringify(incidentRes));
+    }
+
+    function getContentsByAppIdRequest (token, appId, displayFields) {
+        return  '<?xml version="1.0" encoding="UTF-8"?>'+
+        '    <soap:Envelope xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">'+
+        '       <soap:Body>'+
+        '          <ExecuteSearch xmlns="http://archer-tech.com/webservices/">'+
+        '             <sessionToken>' + token + '</sessionToken>'+
+        '         <searchOptions>'+
+        '            <![CDATA[<SearchReport>'+
+        '               <PageSize>100</PageSize>'+
+        '               <MaxRecordCount>100/MaxRecordCount>'+
```

Comment:
and also the closing tag has syntax error.. should be:
`<MaxRecordCount>100</MaxRecordCount>`